_includes/head.html

```diff
@@ -40,11 +40,12 @@
     }
   </script>
   <link rel="stylesheet" href="{{ '/assets/css/style.css' | relative_url }}" />
-  {%- feed_meta -%}
+  <link type="application/atom+xml" rel="alternate" href="{{ '/rss/all.xml' | relative_url }}" title="{{ site.title | escape }}" />
```

Comment:
I don't know Jekyll at all. Can you explain?

Reply:
The line that is being removed is a bunch of magic for generating RSS, which is not the RSS we want. 

The line that is being added is an explicit RSS feed that makes sense for this repo.